alvr/server/cpp/platform/win32/VideoEncoderVCE.cpp

```diff
@@ -224,25 +266,26 @@ amf::AMFComponentPtr VideoEncoderVCE::MakeEncoder(
 		//No noticable performance difference and should improve subjective quality by allocating more bits to smooth areas
 		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_HEVC_ENABLE_VBAQ, true);
 
-		//Fixes rythmic pixelation. I-frames were overcompressed on default settings
+		//Fixes rythmic pixelation.
 		amfEncoder->SetProperty(AMF_VIDEO_ENCODER_HEVC_MAX_QP_I, 30);
```

Comment:
Should look more into turning these as I've seen times when it causes latency to go way too high setting to a higher value seems to fix it.(Zarik said I and P should match so just adding even if the value QP_P is set to is the issue)
https://discord.com/channels/720612397580025886/720639083616927795/1037847999184916491

Reply:
He also said he doesn't think we should be messing with these values anyway.

Reply:
I've tried different value for QP_P, but didn't say much difference. So maybe we shouldn't mess with them (given that we don't have IDR frames anymore).

Reply:
Ok, I'm going to remove them as it seems everyone is in agreement with this. Correct?

Reply:
Hi, 

I noticed huge image quality improvements by setting `m_amfEncoder->SetProperty(AMF_VIDEO_ENCODER_HEVC_MAX_QP_P, 30);` on a 7900 XTX.

I'm using ALXR client from @korejan on a PICO 4 with an outdated server (v18.2.3, the version ALXR works), but I think it could be interesting to let the user mess with these settings on the UI.

I opened this issue on ALXR https://github.com/korejan/ALVR/issues/70